include/Einleitung.tex

```diff
@@ -1,12 +1,12 @@
 \chapter{Einleitung}
-In der kooperativen Speicherbibliothek werden Exemplare der dem Verein zugehörigen Bibliotheken eingelagert. Das heisst das Exemplar der Bibliothek wird gereinigt und inventarisiert eingelagert. Sollte ein Bibliotheksbenutzer ein Exemplar dieses Werks anfordern, wird das Exemplar entweder per Kurier an die jeweilige Bibliothek verschickt, oder bei Journalen oder Magazinen vorzugsweise, eingescannt und in digitaler Form an den Benutzer übergeben. Sowohl die beteiligten Bibliotheken, wie auch die Bibliotheksbenutzer sind daher an einer zeitgerechten Lieferung interessiert. Dies bedeutet für die Speicherbibliothek, dass alle Prozesse und Abläufe effizient und zuverlässig ablaufen müssen. Da Verzögerungen in Zwischenschritten sich auf die ganze Auslieferung des Exemplars oder der digitalen Kopie desjenigen, auswirken.
+In der Kooperativen Speicherbibliothek werden Exemplare der dem Verein zugehörigen Bibliotheken eingelagert. Das heisst, das Exemplar der Bibliothek wird gereinigt und inventarisiert eingelagert. Sollte ein Bibliotheksbenutzer ein Exemplar dieses Werks anfordern, wird das Exemplar per Kurier an die jeweilige Bibliothek verschickt. Journale oder Magazine werden vorzugsweise eingescannt und in digitaler Form an den Benutzer übergeben, sofern dies so vom Benutzer bestellt wurde. Sowohl die beteiligten Bibliotheken, wie auch die Bibliotheksbenutzer sind daher an einer zeitgerechten Lieferung interessiert. Dies bedeutet für die Speicherbibliothek, dass alle Prozesse und Abläufe effizient und zuverlässig ablaufen müssen. Da Verzögerungen in Zwischenschritten sich auf die ganze Auslieferung des Exemplars oder der digitalen Kopie desjenigen, auswirken können.
```

Comment:
Verzögerungen in Zwischenschritten kann sich auf die gesamte Auslieferung des Exemplars, in Physischer oder Digitalen Form, auswirken.